code/drasil-gool/GOOL/Drasil/RendererClasses.hs

```diff
@@ -60,7 +60,7 @@ class (BlockCommentSym r) => RenderFile r where
 
   commentedMod :: SFile r -> FS (r (BlockComment r)) -> SFile r
 
-  fileFromData :: FilePath -> r (Module r) -> SFile r
+  fileFromData :: FilePath -> FSModule r -> SFile r
```

Comment:
I very much approve of making these stylistic changes. However, I would have preferred a few more PRs, each with its own set of smaller changes, rather than bundling these up with a bigger change.